packages/shell-api/bin/api-postprocess.ts

```diff
@@ -3,6 +3,18 @@ import type * as BabelTypes from '@babel/types';
 import { promises as fs } from 'fs';
 import path from 'path';
 import { signatures } from '../';
+import enUs from '../../i18n/src/locales/en_US';
+
+function getHelpClassName(className: string) {
+  switch (className) {
+    case 'CollectionImpl':
```

Comment:
I pointed this out below as well, but this is an example of the kind of fallout from renaming Collection to CollectionImp and reusing the name for a type. And doing the same for Database.

Maybe this is OK but we could also consider leaving Collection and Database unchanged and then just having the types named something else like CollectionWithSchema and DatabaseWithSchema. Or whatever might be a better name.